database-service/tests/test_migrations.py

```diff
@@ -0,0 +1,13 @@
+import sh
+
+def test_migrations_can_be_applied_without_errors(hasura_endpoint):
+    # Given I am in the right location
+    assert 0 == sh.ls('migrations').exit_code 
```

Comment:
This is not how you check if a file exists in Python. First it is not the right tool for the job (use built-in library , see for example https://stackabuse.com/python-check-if-a-file-or-directory-exists/) and it is not efficient since you spawn a new shell in a new process, which involves a lot of overhead.

Reply:
True thing. It's fixed now.